flows/parking/atd_parking_data_meters_txn_history.py

```diff
@@ -88,16 +88,27 @@ def pull_docker_image():
     return
 
 
+# Get the envrioment variables
+@task(
+    name="get_env_vars",
+    state_handlers=[handler],
+    log_stdout=True,
+)
+def get_env_vars():
+    environment_variables = get_key_value(key=f"atd_parking_data_meters")
+    return environment_variables
```

Comment:
👍 🚀 

Just wanted to note this pattern that Charlie has here. It's really helpful, and I'd suggest we follow it for prefect flows going forward unless there's a compelling reason to for a given flow.